stix_shifter_modules/splunk/stix_transmission/api_client.py

```diff
@@ -45,25 +45,26 @@ async def set_splunk_auth_token(self, auth, headers):
         except KeyError as e:
             raise Exception('Authentication error occured while getting auth token: ' + str(e))
 
+
     async def ping_box(self):
-        self.authenticate()
+        await self.authenticate()
         endpoint = self.endpoint_start + 'server/status'
         data = {'output_mode': self.output_mode}
         return await self.client.call_api(endpoint, 'GET', data=data, timeout=self.timeout)
         
     async def create_search(self, query_expression):
         # sends a POST request to 
         # https://<server_ip>:<port>/services/search/jobs
-        self.authenticate()
+        await self.authenticate()
         endpoint = self.endpoint_start + "search/jobs"
         data = {'search': query_expression, 'output_mode': self.output_mode}
         return await self.client.call_api(endpoint, 'POST', data=data, timeout=self.timeout)
 
     async def get_search(self, search_id):
         # sends a GET request to
-        # https://<server_ip>:<port>/services/search/jobs/<search_id>
+        # http]s://<server_ip>:<port>/services/search/jobs/<search_id>
```

Comment:
Was the `]` intentional?

Reply:
Typo error corrected, removed character (]) .